layouts/_default/baseof.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="author" content="{{ $.Site.Author }}" />
+    <meta name="keywords" content="{{ $.Site.Params.keywords }}" />
+    <meta name="description" content="{{ $.Site.Params.description }}" />
+    <meta name="theme-color" content="#fff" />
+
+    <meta property="og:image" content="/img/share-image.png" />
+    <meta property="og:image:width" content="1024" />
+    <meta property="og:image:height" content="512" />
+    <meta property="og:image:alt" content="MapLibre" />
+    <meta property="og:title" content="{{ $.Site.Title }}" />
+    <meta property="og:description" content="{{ $.Site.Params.description }}" />
+    <meta property="og:url" content="{{ $.Site.BaseURL }}" />
+
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@maplibre" />
+    <meta name="twitter:creator" content="@maplibre" />
+    <meta name="twitter:title" content="{{ $.Site.Title }}" />
+    <meta
+      name="twitter:description"
+      content="{{ $.Site.Params.description }}"
+    />
+    <meta name="twitter:image:src" content="/img/share-image.png" />
+    <meta name="twitter:image:alt" content="MapLibre" />
+
+    {{ $style := resources.Get "style.scss" | resources.ToCSS | resources.Minify
+    | resources.Fingerprint }}
+    <link rel="stylesheet" href="{{ $style.RelPermalink }}" />
+    {{ $opts := dict "targetPath" "main.js" }} {{ $built := resources.Get
+    "map-preview.js" | js.Build $opts }}
+    <script type="text/javascript" src="{{ $built.RelPermalink }}"></script>
+
+    <title>{{ $.Site.Title }} - {{ .Title }}</title>
+
+    <!-- Google Tag Manager -->
+    <script>
+      (function (w, d, s, l, i) {
+        w[l] = w[l] || [];
+        w[l].push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != "dataLayer" ? "&l=" + l : "";
+        j.async = true;
+        j.src = "https://www.googletagmanager.com/gtm.js?id=" + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, "script", "dataLayer", "GTM-PRQ9ZC7");
+    </script>
+    <!-- End Google Tag Manager -->
```

Comment:
Can you remind me what this is for?

Reply:
Its some google analytics stuff :D I would favor to exclude this because of data privacy.
Is somebody actively using the data which is collected here?

Reply:
Nobody uses it as far as I know. Feel free to remove this stuff...

Reply:
removed